irohad/torii/impl/command_service_impl.cpp

```diff
@@ -133,7 +134,14 @@ namespace torii {
     const auto &txs = batch->transactions();
     std::for_each(txs.begin(), txs.end(), [this](const auto &tx) {
       const auto &tx_hash = tx->hash();
-      if (cache_->findItem(tx_hash) and tx->quorum() < 2) {
+      auto found = cache_->findItem(tx_hash);
+      if (found
+          and iroha::visit_in_place(
```

Comment:
Please, add comment about why this check about particulary `EnoughSignaturesCollectedResponse` is introduced in that `if`. It's not clear from the context, and future reader of the code will be confused, I think.